lib/internal/streams/duplex.js

```diff
@@ -63,6 +63,24 @@ function Duplex(options) {
   if (!(this instanceof Duplex))
     return new Duplex(options);
 
+  this._events ??= {
```

Comment:
Non-blocking nit: a comment explaining the purpose of defining this like this would be helpful for future generations coming into this code :-)